tst/small.tst

```diff
@@ -30,6 +30,11 @@ gap> notavailable;
   3726, 3744, 3750, 3752, 3760, 3768, 3780, 3784, 3792, 3800, 3808, 3816, 
   3834, 3840, 3848, 3861, 3864, 3872, 3880, 3888, 3912, 3915, 3920, 3936, 
   3942, 3944, 3952, 3960, 3969, 3976, 3984, 3993, 3996, 4000 ]
+gap> notavailable2 := Filtered([1..4000], x -> not SmallGroupsAvailable(x));;
+gap> notavailable = notavailable2;
+true
+gap> Filtered([1..4000], i -> NumberSmallGroupsAvailable(i) <> SmallGroupsAvailable(i));
+[ 1024 ]
```

Comment:
This test *will* break with future extensions of the SGL. Perhaps restrict from 4000 to 2000, the range which the SGL actually promises?